<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SvgDialog</class>
 <widget class="QDialog" name="SvgDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>562</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>SVG Editor</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>520</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>941</width>
     <height>491</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTextEdit" name="svgText"/>
    </item>
    <item>
     <widget class="QLabel" name="svgImage">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="updateButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pickColorButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>520</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Pick Color...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="undoButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>520</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="redoButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>520</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SvgDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SvgDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
